plugins/engines/ospray/OSPRayCamera.cpp

```diff
@@ -21,33 +21,29 @@
 #include "OSPRayCamera.h"
 
 #include <brayns/common/log.h>
+#include <brayns/parameters/ParametersManager.h>
+
 #include <ospray/SDK/common/OSPCommon.h>
 
 namespace brayns
 {
-OSPRayCamera::OSPRayCamera(const CameraType cameraType)
-    : Camera(cameraType)
+OSPRayCamera::OSPRayCamera(ParametersManager& parametersManager)
```

Comment:
const& at least. still quite ugly that a engine-specific camera needs full access to all parameters...

Reply:
I can restrict it to renderingParameters instead. Does this looks better to you? Or same...

Reply:
Also, cannot be const because it can be reinitialized by the camera